src/Execute/executeQuickInput.ts

```diff
@@ -0,0 +1,151 @@
+/*
+ * Copyright (c) 2022 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as vscode from 'vscode';
+import {globalBackendMap} from '../Backend/Backend';
+import { Backend } from '../Backend/API';
+import { MultiStepInput } from '../Utils/MultiStepInput';
+import { exec } from 'child_process';
+import { appendFileSync } from 'fs';
+
+export async function runInferenceQuickInput(context: vscode.ExtensionContext) {
+  interface State {
+    backend: Backend;
+    modelPath: vscode.Uri;
+    inputSpec: string;
+    infer: vscode.QuickPickItem;
+  }
+
+  async function collectInputs() {
+    const state = {} as Partial<State>;
+    await MultiStepInput.run(input => pickBackend(input, state));
+    return state as State;
+  }
+
+  const title = 'Choose Executor Toolchain';
+
+  async function pickBackend(input: MultiStepInput, state: Partial<State>) {
+    const backends: vscode.QuickPickItem[] = Object.keys(globalBackendMap).map(label => ({ label }));
+    state.infer = await input.showQuickPick({
+      title,
+      step: 1,
+      totalSteps: 3,
+      placeholder: 'Pick Executor',
+      items: backends,
+      shouldResume: shouldResume
+    });
+    state.backend = globalBackendMap[state.infer.label];
+    return (input: MultiStepInput) => selectInputModel(input, state);
+  }
+
+  // TODO: Use quickPick window with fast grep child process
+  async function selectInputModel(input: MultiStepInput, state: Partial<State>) {
+    if (state.backend === undefined) {
+      throw Error('Backend is undefined');
+    }
+
+    let backendName: string = state.backend.name();
+    const executor = state.backend.executor();
```

Comment:
if executor is undefined?

Reply:
It will considered by the code of following comment.( https://github.com/Samsung/ONE-vscode/pull/558/files#r864502593)
Thank you for notifying me!

Reply:
I can't access the uri.